### PR TITLE
Adding links to template; specify current link is for metadata.

### PR DESCRIPTION
**Pull request by meg-huggingface**

### CODE REVIEW

File: docs/hub/model-cards.md [hunk 1]
```diff
@@ -42,7 +44,7 @@ The metadata that you add to the model card enables certain interactions on the
 
 Dataset, metric, and language identifiers are those listed on the [Datasets](https://huggingface.co/datasets), [Metrics](https://huggingface.co/metrics) and [Languages](https://huggingface.co/languages) pages and in the [`datasets`](https://github.com/huggingface/datasets) repository.
 
-See the detailed model card specification [here](https://github.com/huggingface/hub-docs/blob/main/modelcard.md?plain=1).
+See the detailed model card metadata [here](https://github.com/huggingface/hub-docs/blob/main/modelcard.md?plain=1).
```
Comment (nateraw) [inline docs/hub/model-cards.md:47]: I think the word specification still nice to have here, but just after "metadata". WDYT?

```suggestion
See the detailed model card metadata specification [here](https://github.com/huggingface/hub-docs/blob/main/modelcard.md?plain=1).
```

**Review by nateraw — COMMENTED** [review]
Thanks! Couple quick changes I thought of

File: docs/hub/model-cards.md (hunk 1 above)
Comment (julien-c) [reply docs/hub/model-cards.md:47]: i like this suggestion

**Review by julien-c — COMMENTED** [review]
looks good to me!

Hopefully after that it's clearer that we encourage authors to input BOTH model card metadata and model card freeform content

**Comment by julien-c** [comment]
are you still working on this, @meg-huggingface @EziOzoani? I think with @nateraw suggestions it would be ready to merge (and would be an improvement vs the current doc) WDYT?

**Comment by EziOzoani** [comment]
> looks good to me!
> 
> Hopefully after that it's clearer that we encourage authors to input BOTH model card metadata and model card freeform content

@julien-c while you're here, could you review the [guidebook on the hub docs](https://github.com/huggingface/hub-docs/pull/545) so the [blog post](https://github.com/huggingface/blog/pull/719) can go out

**Review by julien-c — APPROVED** [review]
ok let's merge this one

**Pull request merged**